### PR TITLE
feat: System Power Key

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR does the following:
- Adds a Power Key to prompt the shutdown prompt
- Fixes this issue: https://github.com/pop-os/keyboard-configurator/issues/94

Tested on a Launch keyboard on firmware version 0.7.103-4007-ga49720.

### CODE REVIEW

**Comment by jacobgkau** [comment]
This PR also increases the width of the `Controls` section of keys from 4 to 5. This does not increase the minimum width of the window, but does make `Controls` the widest section (which was previously `Alphabet keys`.)

Before:
Screenshot from 2021-09-27 13-01-10
Screenshot from 2021-09-27 13-01-39

After:
Screenshot from 2021-09-27 13-04-02
Screenshot from 2021-09-27 13-04-11

Notably, while I do have the unadvertised column width change, I do not see the new `Power` key showing up in the `Controls` section. This is testing on a galp5. Is the key expected to be assignable for laptop keyboards in addition to Launch keyboards?

**Comment by ahoneybun** [comment]
I didn't add it to Laptop keyboards and I only see it when I select the Launch in the Configurator. 

ksnip_20210927-131831

File: layouts/picker.json
Comment (leviport) [inline layouts/picker.json:527]: I believe @jacobgkau is referring to this change. What's the reasoning for it?

**Comment by leviport** [comment]
According to [the QMK docs](https://beta.docs.qmk.fm/using-qmk/simple-keycodes/keycodes), it looks like `KC_POWER` doesn't work on Windows. Should we be exposing a key by default that doesn't work on one of the OSes we build the configurator for?

**Comment by jacobgkau** [comment]
> According to [the QMK docs](https://beta.docs.qmk.fm/using-qmk/simple-keycodes/keycodes), it looks like `KC_POWER` doesn't work on Windows. Should we be exposing a key by default that doesn't work on one of the OSes we build the configurator for?

From that list and some other docs, it looks like `KC_PWR` would be an alternative that Windows would recognize?

File: layouts/picker.json
Comment (ahoneybun) [reply layouts/picker.json:527]: It didn't align the button correctly without making that change. It would be like this:

| Screen Up | Screen Down |
| Suspend |
| Power |

Instead of 

| Screen Up | Screen Down |
| Suspend | Power |

**Comment by ahoneybun** [comment]
It looks like `KC_PWR` will work on Linux as well, not sure about macOS.

**Comment by leviport** [comment]
> It looks like `KC_PWR` will work on Linux as well, not sure about macOS.

Looks like it checks all 3 columns to me:

**Comment by ahoneybun** [comment]
I just updated it to use 'KP_PWR' instead.

File: layouts.py
Comment (ids1024) [inline layouts.py:46]: The `KP_` keycodes are for the numeric keypad, and QMK doesn't seem to have a `KP_PWR` (i.e. what would be `KC_KP_PWR` in QMK's source; the prefix is stripped here).

Just removing this lines should be fine. I don't think `layout.py` requires any changes.

Comment (ids1024) [reply layouts.py:46]: `layouts.py` isn't actually used during build of the Configurator, it's used to generate the layouts files that are committed to the repository.

This branch is still sending the `POWER` keycode, since that's what's set in `picker.json`. You want to change that to `SYSTEM_POWER` (which `PWR` is an alias for), and remove the change in `layouts.py`.

Comment (ahoneybun) [reply layouts.py]: I've made that change though I don't have my Launch to test this right now.

**Review by ids1024 — DISMISSED** [review]
Seems to work. Should be good now, assuming we're happy with exposing it in this way.

File: layouts.py
Comment (leviport) [reply layouts.py]: Works on my Launch

File: layouts/picker.json
Comment (leviport) [reply layouts/picker.json:527]: @ids1024 any thoughts on this? It seems weird to make a change that affects the laptop keyboard control button group in order to better arrange the Launch control button group. It doesn't seem to break anything though, so I'm not sure I have a huge problem with it.

**Comment by reesericci** [comment]
whats stopping this PR from being merged?

**Comment by isantop** [comment]
> whats stopping this PR from being merged?

All code in this organization is required to pass QA testing before being merged. It's currently awaiting QA Approval.

File: layouts/picker.json
Comment (leviport) [reply layouts/picker.json:527]: @ahoneybun can you remove this change? It should be fixed by https://github.com/pop-os/keyboard-configurator/pull/98

Once that's done, I'm ready to approve this PR.

Comment (ahoneybun) [reply layouts/picker.json]: Should be changed now though we need @ids1024 to review it again now.

Comment (leviport) [reply layouts/picker.json]: Thanks! I'll have one last look at it soon.

**Review by ids1024 — APPROVED** [review]

**Review by leviport — APPROVED** [review]
Other than the layout bug (which should be fixed by https://github.com/pop-os/keyboard-configurator/pull/98), this is looking good.

**Pull request merged**
